google/resource_compute_region_instance_group_manager.go

```diff
@@ -509,3 +529,41 @@ func resourceComputeRegionInstanceGroupManagerDelete(d *schema.ResourceData, met
 	d.SetId("")
 	return nil
 }
+
+func expandDistributionPolicy(configured *schema.Set) *computeBeta.DistributionPolicy {
+	if configured.Len() == 0 {
+		return nil
+	}
+
+	distributionPolicyZoneConfigs := make([]*computeBeta.DistributionPolicyZoneConfiguration, 0, configured.Len())
+	for _, raw := range configured.List() {
+		data := raw.(string)
+		distributionPolicyZoneConfig := computeBeta.DistributionPolicyZoneConfiguration{
+			Zone: "zones/" + data,
+		}
+
+		distributionPolicyZoneConfigs = append(distributionPolicyZoneConfigs, &distributionPolicyZoneConfig)
+	}
+	return &computeBeta.DistributionPolicy{Zones: distributionPolicyZoneConfigs}
+}
+
+func flattenDistributionPolicy(distributionPolicy *computeBeta.DistributionPolicy) *schema.Set {
+	zones := make([]interface{}, 0)
+
+	if distributionPolicy != nil {
+		for _, zone := range distributionPolicy.Zones {
+			zones = append(zones, zone.Zone)
+		}
+	}
+
+	return schema.NewSet(schema.HashSchema(&schema.Schema{
```

Comment:
I think just `return zones` will work (and then update the return type)

Reply:
FWIW, I tried this, to no avail.

Reply:
I just uploaded a commit where this seems to work. Let me know what you think!